simularium_metrics_calculator/metrics_registry.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from typing import Dict
+
+from .calculators import (
+    AgentsCalculator,
+    NearestNeighborCalculator,
+    NumberOfAgentsCalculator,
+    TimesCalculator,
+)
+from .constants import METRIC_TYPE
+from .exceptions import MetricNotFoundError
+from .metric_info import MetricInfo
+
+metrics_list = [
+    MetricInfo(
+        uid=0,
+        display_name="Time",
+        metric_type=METRIC_TYPE.PER_TIME,
+        calculator=TimesCalculator,
+    ),
+    MetricInfo(
+        uid=1,
+        display_name="Agent IDs",
+        metric_type=METRIC_TYPE.PER_AGENT,
+        calculator=AgentsCalculator,
+    ),
+    MetricInfo(
+        uid=2,
+        display_name="Number of agents",
+        metric_type=METRIC_TYPE.PER_TIME,
+        calculator=NumberOfAgentsCalculator,
+    ),
+    MetricInfo(
+        uid=3,
+        display_name="Nearest neighbor distance",
+        metric_type=METRIC_TYPE.PER_AGENT,
+        calculator=NearestNeighborCalculator,
+    ),
+]
+
+
+def create_metrics_registry() -> Dict[int, MetricInfo]:
+    """
+    Get a dict mapping metric unique ID to
+    info about each available metric.
+    """
+    result = {}
+    for metric in metrics_list:
+        result[metric.uid] = metric
+    return result
+
+
+metrics_registry = create_metrics_registry()
```

Comment:
I think you explained it before, but I don't understand why the metrics_manager can't make this call as a property on the class: 
```
class MetricsManger():
     metrics_registry = create_metrics_registry()
```
and then call it by using `MetricsManger.metrics_registry`. It just seems like where it should live

Same with the function below (being on the metrics manager)

Reply:
I basically agree with the above comment.   It's often better to have fewer global variables and more variables encapsulated as class members.  I'd even force MetricsManager to have no static methods and have any app using this just instantiate one MetricsManager at startup.  And part of the init of the MetricsManager is to set up the registered metrics.

Reply:
Will this work so that I don't have to instance `MetricsManager` or create this dict every time it's used? I thought since it's a static property it couldn't be set to a function return value like that

Reply:
ok, I made it so you have to instantiate `MetricsManager` at start, it no longer has any static methods, and it sets up the registry at init.